plugins/uefi/README.md

```diff
@@ -11,6 +11,12 @@ With the UpdateCapsule boot service it can be used to update system firmware.
 If you don't want or need this functionality you can use the
 `-Dplugin_uefi=false` option.
 
+Firmware Format
+---------------
+
+The daemon will decompress the cabinet archive and extract a firmare blob in
+EFI executable (capsule) file format.
```

Comment:
Technically this is not executable

Reply:
`blob in an EFI Capsule file format`? Is Capsule a file format?

Reply:
Well the structure of a capsule is defined by the EFI specs.
Section 22.2.2: " DEFINED FIRMWARE MANAGEMENT PROTOCOL DATA
CAPSULE STRUCTURE"
https://www.uefi.org/sites/default/files/resources/UEFI%20Spec%202_6.pdf